### ISSUE TITLE
Compiling 1.2 on Ubuntu 24.04

### ISSUE THREAD

**Issue by altert** [ruven/iipsrv#269]
Compiling version 1.2 on Ubuntu 24.04 results in errors such as:
`RawTile.h:29:1: note: ‘uint32_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?`

This is fixed in current master by `#include <cstdint>` but as the latest release is 1.2, maybe it would be better to include it there as well?

**Comment by ruven** [comment]
Yes, I could do a point release based on 1.2 or maybe better bring forward a new 1.3 release including the new features within the current master.
